### PR TITLE
Update 8.10.3.asciidoc

### PR DESCRIPTION
**Pull request by bmorelli25**
Removes coming in 8.10.3 tag

### CODE REVIEW

**Review by abdonpijpelink — APPROVED** [review]
LGTM

File: docs/reference/release-notes/8.10.3.asciidoc [hunk 1]
```diff
@@ -1,8 +1,6 @@
 [[release-notes-8.10.3]]
 == {es} version 8.10.3
```
Comment (DaveCTurner) [inline docs/reference/release-notes/8.10.3.asciidoc:3]: NB this release has the same known issue as previous 8.10.x releases, we'll need to add this (either now or in a future PR):

```suggestion

[[known-issues-8.10.3]]
[float]
=== Known issues

include::8.10.0.asciidoc[tag=repositorydata-format-change]
```

Comment (bmorelli25) [reply docs/reference/release-notes/8.10.3.asciidoc:3]: thank you

File: docs/reference/release-notes/8.10.3.asciidoc [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 
-coming[8.10.3]
+[[known-issues-8.10.3]]
+[float]
+=== Known issues
 
+include::8.10.0.asciidoc[tag=repositorydata-format-change]
```
Comment (bmorelli25) [inline docs/reference/release-notes/8.10.3.asciidoc:8]:
```suggestion
include::8.10.0.asciidoc[tag=repositorydata-format-change]

```

**Comment by bmorelli25** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|main|[](https://github.com/elastic/elasticsearch/pull/100614)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**